### DISCUSSION TITLE
`client` repo > `Update.js` > `name`

### DISCUSSION THREAD

**Discussion by hens-virtuella-varld** [data-interaction-fed24/server#13] [Q&A] [answered]
In `Update.js`, in return, `value={name}`, does it mean the stuff in `value` is `{name: <value>}`, and later on we store in `body: JSON.stringify({ name }),` ?

**Comment by Liam-hi** [comment, marked as answer]
Exactly! Whenever the user writes any name in the input field, we will retrieve the name through `value={name}`. As you can see, we created a useState hook that creates a state variable "name" and a function "setName" that updates the state. So when we write for instance "Kevin" in the input field, the function handleChange will be called. The handleChange function will then update the name to "Kevin" 

The body sends the name Kevin to the server and the reason why we have `body: JSON.stringify({ name })` is because we want to convert the name to a JSON string.

**Reply by hens-virtuella-varld** [reply]
Thank you for the very clear answer!🌸❤️
